src/main/java/Schedule.java

```diff
@@ -21,7 +21,7 @@
     public Schedule(Data info)
     {
         this.info = info;
-        possibleClassList = new ArrayList<PossibleClass>(info.getNumberOfClasses());
+        //possibleClassList = new ArrayList<PossibleClass>(info.getNumberOfClasses());
```

Comment:
Did you mean to comment all of these or were they just temporary comments that you forgot to uncomment?

Reply:
They are just temporarily commented to allow it to be run until the inputs are finished